google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStubSettings.java

```diff
@@ -456,6 +456,12 @@ public BigtableInstanceAdminStub createStub() throws IOException {
             "Transport not supported: %s", getTransportChannelProvider().getTransportName()));
   }
 
+  /** Returns the default service name. */
+  @Override
+  public String getServiceName() {
+    return "bigtableadmin";
+  }
```

Comment:
Java compiler is complaining 

```
Error:  /home/runner/work/java-bigtable/java-bigtable/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableTableAdminStubSettings.java:[548,3] method does not override or implement a method from a supertype
```

Reply:
I recently added `getServiceName()` to StubSettings. I think we'll need to update shared-deps to the latest version.

Reply:
Thanks. Added the latest shared dependencies BOM and the error disappeared.